rpc_jobs/defaults.yml

```diff
@@ -8,6 +8,7 @@
     DEPLOY_SWIFT: "yes"
     DEPLOY_CEPH: "no"
     DEPLOY_ELK: "yes"
+    DEPLOY_SUPPORT_ROLE: "yes"
```

Comment:
The idea here is to actually test the support role in periodic jobs.  I'm fine not testing it in PRs, but I think it's beneficial to at least test periodically.